genie-web/src/test/groovy/com/netflix/genie/web/spring/autoconfigure/scripts/ScriptsAutoConfigurationSpec.groovy

```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ *  Copyright 2019 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.web.spring.autoconfigure.scripts
+
+import com.netflix.genie.web.properties.ClusterLoadBalancerScriptProperties
+import com.netflix.genie.web.properties.ScriptManagerProperties
+import com.netflix.genie.web.scripts.ScriptManager
+import io.micrometer.core.instrument.MeterRegistry
+import org.springframework.core.io.ResourceLoader
+import org.springframework.scheduling.TaskScheduler
+import spock.lang.Specification
+
+class ScriptsAutoConfigurationSpec extends Specification {
```

Comment:
There is a much better way to test this which actually tests it in a spring environment. You can look at some of my other test classes for AutoConfigurations

Reply:
👍